examples/bert/bert_train.py

```diff
@@ -386,6 +385,18 @@ def main(_):
 
     model_config = MODEL_CONFIGS[FLAGS.model_size]
 
+    if tf.config.list_logical_devices("TPU"):
+        # Connect to TPU and create TPU strategy.
+        resolver = tf.distribute.cluster_resolver.TPUClusterResolver(
```

Comment:
I think we can replace the next few lines with

`resolver = tf.distribute.cluster_resolver.TPUClusterResolver.connect(tpu='local')`

Reply:
done!